kvmagent/kvmagent/plugins/prometheus.py

```
@@ -1564,6 +1565,177 @@ def collect_tianshu_gpu_status():
+        for gpuinfo in gpuinfos:
+            if gpuinfo["pciAddress"] == dev_bus_id:
+                gpuinfo["rxPciInBytes"] = rx
+                gpuinfo["txPciInBytes"] = tx
+                break
+
+    return gpuinfos
+
+
+def collect_vastai_gpu_status():
+    def extract_number(s):
+        if s is None or s == "": return None
```

Ruff (0.8.2)

Multiple statements on one line (colon)

(E701)